src/optimize-streams.ts

```diff
@@ -119,17 +110,25 @@ export class GenericOptimizeTransform extends Transform {
  * Transpile JavaScript to ES5 using Babel.
  */
 export class JSCompileTransform extends GenericOptimizeTransform {
-  constructor() {
-    const transformer = (content: string, file: File) =>
-        babelTransform(content, {
-          presets: [babelPresetES2015NoModules],
-          plugins: [
+  constructor(options: OptimizeOptions['js']) {
+    const transformer = (content: string, file: File) => {
+      const presets = [];
+      const plugins = [
```

Comment:
add comment explaining why we're loading sync plugins without the transform. Also, make these a single array to use with the minify transform.

Reply:
Done.